### PR TITLE
Replaced linux .config with siklu_277_v2_defconfig

### PR DESCRIPTION
**Pull request by yba-siklu**
## Description
Replaced linux .config with siklu_PCB277_V2_defconfig and changed build.sh
appropriately. Also added -eu and other tweaks tioio build.sh.

## Tests
Used "make savedefconfig" to get a defconfig file. Then moved the defconfig file to arch/arm/configs/siklu_PCB277_V2_defconfig, Then retested the build using make siklu_PCB277_V2_defconfig, then did a meld on the old .config and the new .config.

## Jira Issues
JIRA:PORTF-849

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+#!/bin/bash -eu
 
 PATH=/opt/arm/gcc-linaro-7.5.0-2019.12-i686_arm-linux-gnueabihf/bin:$PATH
 export INSTALL_MOD_PATH=$(pwd)/modules
 export ARCH=arm
 export CROSS_COMPILE=arm-linux-gnueabihf-
 
-make menuconfig
+#make savedefconfig
+#make menuconfig
```
Comment (dima1308) [inline build.sh:9]: Can you remove commented out lines?

**Review by dima1308 — COMMENTED** [review]
Is it the same config as was before using menuconfig?

File: build.sh (hunk 1 above)
Comment (yba-siklu) [reply build.sh:9]: No. All of the commented lines in this file need to remain in the file. This is a template file that should only be edited locally and never re-committed (except when I do so by mistake). The purpose of this file is to give you a simple template for building.

**Comment by yba-siklu** [comment]
> Is it the same config as was before using menuconfig?

I don't understand the question.

**Comment by dima1308** [comment]
> I don't understand the question.

Did you change the configuration parameters or just moved the exactly same configuration as we use today to the siklu_PCB277_V2_defconfig file?

**Comment by yba-siklu** [comment]
> > I don't understand the question.
> 
> Did you change the configuration parameters or just moved the exactly same configuration as we use today to the siklu_PCB277_V2_defconfig file?

I used "make savedefconfig" to get a defconfig file. Then moved the defconfig file to arch/arm/configs/siklu_PCB277_V2_defconfig, Then retested the build using make siklu_PCB277_V2_defconfig, then did a meld on the old .config and the new .config.

**Review by dima1308 — APPROVED** [review]

**Review by baruchsiach — APPROVED** [review]

**Pull request merged**